src/index.js

```diff
@@ -1,78 +1,73 @@
-      <i class="fas fa-ellipsis-v"></i>
-      `;
-      li.childNodes[0].addEventListener('change', () => changeState(li.childNodes[0], task.index, this.listObj));
-      return listContainer.append(li);
+    const inputField = document.querySelector('.input-container');
+    const clearSelectedBtn = document.getElementById('clear-selected');
+    inputField.addEventListener('submit', (e) => {
+      e.preventDefault();
+      return add(this.listObj);
     });
+    clearSelectedBtn.addEventListener('click', () => clearSelected(this.listObj));
+    if (this.listObj.length) {
+      this.listObj.forEach((task, index) => {
+        const li = document.createElement('li');
+        li.id = index + 1;
+        this.listObj[index].index = index + 1;
+        li.className = 'list_item';
+        // checkbox
+        const checkBox = document.createElement('input');
+        checkBox.type = 'checkbox';
+        // description
+        const desc = document.createElement('input');
+        desc.className = 'desc';
+        desc.type = 'text';
+        li.append(checkBox, desc);
+        // Icon
+        li.innerHTML += `
+        <i class="fas fa-ellipsis-v" id="move-icon"></i>
+        <i class="fas fa-trash d-none" id="trash-icon"></i>
+        `;
+        li.childNodes[1].value = task.description;
+        li.childNodes[0].addEventListener('change', () => changeState(li.childNodes[0], task.index, this.listObj));
+        li.childNodes[1].addEventListener('click', () => {
+          li.classList.add('editing-state');
+          li.childNodes[3].classList.add('d-none');
+          li.childNodes[5].classList.remove('d-none');
+        });
+        li.childNodes[1].addEventListener('blur', () => {
+          li.classList.remove('editing-state');
+          setTimeout(() => {
+            li.childNodes[3].classList.remove('d-none');
+            li.childNodes[5].classList.add('d-none');
+          }, 100);
+        });
+        li.childNodes[1].addEventListener('input', () => edit(li.childNodes[1], task.index, this.listObj));
+        li.childNodes[5].addEventListener('click', () => remove(this.listObj, task.index));
+        return listContainer.append(li);
+      });
+    } else {
+      const emptyList = document.createElement('h2');
+      emptyList.className = 'empty-list';
+      emptyList.textContent = 'Add Something today';
+      listContainer.append(emptyList);
+    }
   }
 }
 
 const mainListContainer = document.getElementById('list');
 const list = new List();
-list.createItems(mainListContainer);
-
-export default list;
-/*
-  What should I do :
-  1) structure
-    ** Array of objects (list)
-      ==> description (string)
-      ==> completed (boolean)
-      ==> index (num starts from 1)
-    ** class
-      ==> interate the array
-      ==> Save to local storage
-  2) Ineractive
-    ** Change content
-    ** Drag & Drop
-    ** Add & Remove
-    ** Remove All
-*/
\ No newline at end of file
+list.createItems(mainListContainer);
```

Comment:
- [ ] On page load your app doesn't keep the state stored in the localStorage. When a task is marked as completed, it will be unmarked after a page refresh. It is likely due to the fact that you are have not implemented an on-load event with your values stored in the localStorage.